first-bot/adding-a-config-file.md

```diff
@@ -76,10 +76,10 @@ We're using `prefix` in a few places, so we need to change them all. Here's how
 client.on("message", (message) => {
```

Comment:
```suggestion
client.on("messageCreate", (message) => {
```

Reply:
Fixed in b5c8ebfc63f03855f21e383c7d0c06ddeea78255